src/d3a/d3a_core/simulation.py

```diff
@@ -72,24 +71,20 @@ def __init__(self, setup_module_name: str, simulation_config: SimulationConfig =
         )
 
         self.simulation_config = simulation_config
-        self.use_repl = use_repl
-        self.export_on_finish = export
+        self.use_repl = repl
+        self.export_on_finish = not no_export
         self.export_path = export_path
+        if export_subdir is None:
```

Comment:
If the export_subdir is set to something other than None, there is no 'self.export_subdir = export_subdir' line, therefore its value would be lost. Probably I am not seeing something here though.

Reply:
No you are right! Adding it!